packages/foam-vscode/src/features/preview/wikilink-embed.ts

```diff
@@ -123,4 +121,60 @@ function withLinksRelativeToWorkspaceRoot(
   return text;
 }
 
+export function retrieveNoteConfig(): {
+  noteScope: string;
+  noteStyle: string;
+} {
+  let config = getFoamVsCodeConfig<string>(CONFIG_EMBED_NOTE_TYPE); // ex. full-inline
+  let [noteScope, noteStyle] = config.split('-');
+
+  // **DEPRECATED** setting to be removed
+  // for now it overrides the above to preserve user settings if they have it set
+  if (getFoamVsCodeConfig<boolean>(CONFIG_EMBED_NOTE_IN_CONTAINER, false)) {
```

Comment:
From this I assume that if someone has both settings, the old setting will overwrite the new one?
I think that the new setting should always have the precedence, and we should only read the previous one if the new setting has not been used, wdyt?